CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ set(TFLITE_SOURCE_DIR "${TENSORFLOW_SOURCE_DIR}/tensorflow/lite")
 add_subdirectory("${TFLITE_SOURCE_DIR}" "${CMAKE_CURRENT_BINARY_DIR}/tensorflow-lite" EXCLUDE_FROM_ALL)
 
 # Generic compilation options and settings
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Side question: does this mean anything for our CI builds and releases in the `bboxpi` repository? There we do have to make sure that we can link with old compilers for example.

[Here](https://github.com/plumerai/bboxpi/blob/main/.github/workflows/cpp.yml) we build with GCC 8, which should be fine according to https://gcc.gnu.org/projects/cxx-status.html#cxx17

So I guess it is all fine but just double-checking.